### ISSUE TITLE
Use mapURI to provide an exhaustive list of example URIs for certain LDA methods

### ISSUE THREAD

**Issue by egonw** [openphacts/OPS_LinkedDataApi#14]
The current LDA documentation often has an example URIs for a compound, protein, or gene, but there is some use for more examples to give people an indication what database-style URIs they can use. The reference example from the IMS telcon today is the use of Wikipedia URLs for compounds.

The idea is to add the following text to the appropriate LDA documentation:

    More example URIs are found [here].

The [here] could link to a page listing more examples. In reply to using a support.ops.org page, the suggestion was made to use a generated list. Such lists are already generated at two places:

* the LDA's /mapUri (with HTML output format, which does not exist yet)
* the IMS directly

But the production IMS is not public so it would not have the advantage of the first that it always be consistent (because it uses the exact same API version and thus data).

**Comment by danidi** [comment]
The IMS link would have the advantage that no API keys are necessary. A disadvantage of both would be that the link is rather long to display in the documentation. Not sure if is is possible to only show "here" and make it an actual link on the doc page?

**Comment by egonw** [comment]
@danidi agreed... [here] should show visually just "here" and the hyperlink should be hidden. If that is technically possible needs to be explored. This was basically my question to Antonis... we'll figure that out. Possibly, we use a URL shortener :)
